lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -177,7 +177,7 @@
     - List of dictionaries describing the service configs.
     - Every item must be a dictionary exposing the keys config_id, config_name, filename, uid (defaults to 0), gid (defaults to 0), mode (defaults to 0o444)
     - Maps docker service --config option.
-    default: []
+    default: null
```

Comment:
How about just leaving it away (and removing `default=None` in args_spec)? The default value is `None` anyway if the argument is not required.

Reply:
@felixfontein  Thank you, I didn't know that; I'm gonna change them